app/utils/csv.py

```diff
@@ -78,21 +78,21 @@ def generate_notifications_csv(**kwargs):
             template=get_sample_template(kwargs["template_type"]),
         )
         original_column_headers = original_upload.column_headers
-        fieldnames = (
-            ["Row number"]
-            + original_column_headers
-            + [
-                "Template",
-                "Type",
-                "Sent by",
-                "Job",
-                "Carrier",
-                "Carrier Response",
-                "Status",
-                "Time",
-            ]
-        )
+        fieldnames = [
+            "Template",
+            "Type",
+            "Sent by",
+            "Job",
+            "Carrier",
```

Comment:
After our discussion with AWS yesterday, let's remove the Carrier field from the CSV reports for now:

```suggestion
```